infracost-usage-example.yml

```
@@ -502,6 +502,33 @@ resource_usage:
     monthly_analytical_storage_write_operations: 1000000 # Monthly number of write analytical storage operations.
     monthly_analytical_storage_read_operations: 1000000 # Monthly number of read analytical storage operations.
     max_request_units_utilization_percentage: 50 # Average utilisation of the maximum RU/s, starting at 10%. Possible values from 10 to 100.
+  
+  azurerm_dns_a_record.my_a_record:
+    monthly_queries: 10000
```

We using full-number of cost parameters in usage-file. You mean here (and in all the rest of azurerm_dns* resources below) that it is 10000 millions of queries, but it can be not pretty obvious.

```suggestion
  azurerm_dns_a_record.my_a_record:
    monthly_queries: 1500000000  # Monthly number of DNS queries
```